pkgs/development/libraries/applet-window-appmenu/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, kcoreaddons
+, kdeclarative
+, kdecoration
+, plasma-framework
+, plasma-workspace
+, libSM
+, qtx11extras
+, kwindowsystem
+, libdbusmenu
+}:
+
+mkDerivation rec {
```

Comment:
Could you try using `stdenv.mkDerivation` here? #180841

Reply:
It doesn't build that way.
```
qtPreHook
Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
error: builder for '/nix/store/gk00f0zyxbr9rc3hnrmfk2da5isaby6r-applet-window-appmenu-unstable-2022-06-27.drv' failed with exit code 1;
       last 2 log lines:
       > qtPreHook
       > Error: wrapQtAppsHook is not used, and dontWrapQtApps is not set.
       For full logs, run 'nix log /nix/store/gk00f0zyxbr9rc3hnrmfk2da5isaby6r-applet-window-appmenu-unstable-2022-06-27.drv'.
```